examples/model_compress/pruning/basic_pruners_torch.py

```diff
@@ -0,0 +1,347 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+'''
+Examples for basic pruners
+'''
```

Comment:
what is the difference between this file and `basic_pruners_kd_torch.py`

Reply:
simplify the kd example, please review the latest version.